modules/local/sentieon/datametrics.nf

```diff
@@ -0,0 +1,62 @@
+process SENTIEON_DATAMETRICS {
+    tag "$meta.id"
+    label 'process_high'
+    label 'sentieon'
+
+    input:
+    tuple val(meta), path(bam), path(bai)
+    path fasta
+    path fai
+
+    output:
+    tuple val(meta), path('*mq_metrics.txt')   , emit: mq_metrics
+    tuple val(meta), path('*qd_metrics.txt')   , emit: qd_metrics
+    tuple val(meta), path('*gc_summary.txt')   , emit: gc_summary
+    tuple val(meta), path('*gc_metrics.txt')   , emit: gc_metrics
+    tuple val(meta), path('*aln_metrics.txt')  , emit: aln_metrics
+    tuple val(meta), path('*is_metrics.txt')   , emit: is_metrics
+    path  "versions.yml"                       , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args   = task.ext.args ?: ''
+    def input  = bam.sort().collect{"-i $it"}.join(' ')
+    def ref    = fasta ? "-r $fasta" : ''
+    def prefix       = task.ext.prefix ?: "${meta.id}"
+    """
+    sentieon \\
+        driver \\
+        -t $task.cpus \\
+        -r $fasta \\
```

Comment:
```suggestion
        $ref \\
```
Looks like you define ref further up. Alternatively remove the `def ref` on line 26